.github/workflows/publish.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
   build_and_publish:
 
-    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish.yml@28e947497bed4d6ec3fa1d66d198e95a1d17bc63  # v2.2.1
```

Comment:
note that this hash *doesn't* match the version in comment, but points ahead of it, because version 2.2.3 isn't released yet.

https://github.com/OpenAstronomy/github-actions-workflows

(pyerfa maintainers: this class of issue can be treated as security problems and as such, can be linted against using zizmor. Let me know if you're interested and I can set it up for this repo, as I previously did for astropy)

Reply:
Sorry about that! @neutrinoceros should we wait for a new release of OpenAstronomy/github-actions-workflows and then bump the hash/version to that v2.2.3 release?

Reply:
2.3.0 is out